extensions/google-cloud/storage/gcs_event_store/gcs_event_store_test.go

```diff
@@ -34,6 +34,26 @@ func TestGCSEventStore(t *testing.T) {
 	_ = os.Setenv("STORAGE_EMULATOR_HOST", "localhost:19081")
 	folderName := "folder-name"
 
+	t.Run("query empty bucket", func(t *testing.T) {
+		bucket := "empty-bucket"
+		setup(t, bucket)
+		config := gcs_event_store.Config(bucket).WithFolder(folderName)
+		eventStore, err := gcs_event_store.New(context.TODO(), config, option.WithoutAuthentication())
+		assert.NoError(t, err)
+
+		sources, err := eventStore.ListSourcesByKey(context.TODO(), key)
+		assert.NoError(t, err)
+		assert.ElementsMatch(t, []string{}, sources)
```

Comment:
It seems that the elements returned could be not in lexicographic order. Not sure if it's caused by the `oittaa/gcp-storage-emulator` implementation.

I'll just change it back to not check the order in this test, since it's SLA of GCS, not this lib.